.github/workflows/conformance-ginkgo.yaml

```diff
@@ -211,8 +221,61 @@ jobs:
             until docker manifest inspect quay.io/${{ env.QUAY_ORGANIZATION_DEV }}/$image:${{ needs.check_changes.outputs.sha }} &> /dev/null; do sleep 45s; done
           done
 
+  generate-matrix:
+    name: Generate Job Matrix from YAMLs
+    needs: check_changes
+    runs-on: ubuntu-latest
+    outputs:
+      matrix: ${{ steps.set-matrix.outputs.matrix }}
+    steps:
+      - name: Checkout PR code
+        uses: actions/checkout@8e5e7e5ab8b370d6c329ec480221332ada57f0ab # v3.5.2
+        with:
+          ref: ${{ needs.check_changes.outputs.matrix_sha }}
+          persist-credentials: false
+
+      - name: Convert YAML to JSON
```

Comment:
Is this done because we need `jq`-specific features that `yq` is lacking?

Reply:
@nebril no, it's because of the there isn't a `fromYaml` function, similar to the `fromJson` on GitHub as documented https://docs.github.com/en/enterprise-cloud@latest/actions/learn-github-actions/expressions#fromjson